config-connector/solutions/projects/kpt/simple-project/Kptfile

```diff
@@ -0,0 +1,36 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: simple-project
+packageMetadata:
+  shortDescription: A simple project
+openAPI:
+  definitions:
+    io.k8s.cli.setters.org-id:
+      description: organization ID for associated services
+      x-k8s-cli:
+        setter:
+          name: org-id
+          value: "${ORG_ID?}"
+          setBy: PLACEHOLDER
+    io.k8s.cli.setters.project-id:
+      description: ID of project
+      x-k8s-cli:
+        setter:
+          name: project-id
+          value: simple-project
+          setBy: package-default
+    io.k8s.cli.setters.project-name:
```

Comment:
Can we default this to be the same as project-id?

Reply:
Yes, currently they both have value of `simple-project`. Should I make them use the same setter and therefore always have the same value?

Reply:
The ideal would be that setting `project-id` will set both, while setting `project-name` would override the project name only.

Reply:
The best way I can think of to implement this behavior with kpt would be to have them both be set by the project-id setter and make a comment in project.yaml noting that the spec.name can be changed independently of the metadata.name.

Reply:
Could we ask kpt team about multi-setters or something like this? It seems like a reasonable use case.

Reply:
I have created an issue for it [here](https://github.com/GoogleContainerTools/kpt/issues/430).

Reply:
@morgante in the mean time I have made them both use the same setter. Please take a look when you get the chance.